ResearchKit/Consent/ORKConsentSignatureFormatter.h

```diff
@@ -0,0 +1,8 @@
+#import <ResearchKit/ResearchKit.h>
+#import <ResearchKit/ORKConsentSignature.h>
+
+@interface ORKConsentSignatureFormatter : NSObject
+
+- (NSString *)HTMLForSignature:(ORKConsentSignature *)signature;
```

Comment:
I made them instance methods for two reasons:

1) Habit. At Pivotal, we use [Cedar](https://github.com/pivotal/cedar) for testing Objective-C code, and while Cedar has very rich support for mocks out of the box, it doesn't support mocking class methods. Of course here, I'm hand-rolling the mocks anyway, so this wouldn't be an issue.

2) It more closely follows the pattern established by Foundation formatter classes such as `NSDateFormatter` and `NSNumberFormatter`. Those classes feature state while the formatters I've introduced do not, but I thought that in terms of maintaining a consistent API, it was worth making them instance methods.